pipelines/skylab/multiome/atac.wdl

```diff
@@ -265,13 +267,26 @@ task AddCBtags {
   input {
     File bam
     String output_base_name
-    Int disk_size = ceil(size(bam, "GiB") + 50)
-    Int mem_size = 10
+    Int disk_size = ceil(size(bam, "GiB") + 500)
+    Int mem_size = 30
+    
+    # script for monitoring tasks
+    File monitoring_script
```

Comment:
After testing, we will probably want to take out the monitoring script for the release.

Reply:
Good point!